app/views/groups/show.html.haml

```diff
@@ -22,10 +22,10 @@
     %header
       .row
         .small-12.columns
-          = image_tag @group.promo_image.variant(resize_to_limit: [1200, 260]) if @group.promo_image.variable?
+          = image_tag @group.promo_image.variant(resize_to_limit: [1200, 260]) if @group.promo_image.attached?
```

Comment:
I chose `variable?` because it's more specific than `attached?`. A file can be attached but not variable and then it raises an error on calling `variant`. If the file is not attached then it's also not variable. Is there a reproducible test case where `attached?` is better?

I checked the example from fr-staging and the image is attached and variable but when downloading the image to process the variant it raises an IntegrityError. The image file is corrupt. Our `url_for` method then returns `nil` which raises an error when passed to `image_tag`. So maybe we need a new helper: `image_tag_if_present` which guards against nil.